corehq/apps/locations/models.py

```diff
@@ -55,6 +58,14 @@ def root_locations(cls, domain, include_archive_ancestors=False):
         roots = cls.objects.root_nodes().filter(domain=domain)
         return _filter_for_archived(roots, include_archive_ancestors)
 
+    @memoized
+    def get_products(self):
+        """
+        If there are no products specified for this location, assume all
+        products for the domain are relevant.
+        """
+        return self.products.all() or SQLProduct.by_domain(self.domain)
```

Comment:
Not in this instance.  The UI uses loc.products directly (not this wrapper).  Kinda clumsy to have two different accessors, I know, but I didn't see another decent way.  Happy to hear alternatives if you've got 'em!